package.json

```diff
@@ -54,33 +57,33 @@
   ],
   "dependencies": {
     "@typescript-eslint/parser": "^5.0.0",
-    "eslint-config-standard": "^16.0.0"
+    "eslint-config-standard": "^17.0.0-1"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.0.1 || ^5.0.0",
-    "eslint": "^7.12.1",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1 || ^5.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "eslint": "^8.0.1",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-n": "^14.0.0",
```

Comment:
Is this the earliest release we are able to require?

Reply:
Yes, see: https://github.com/standard/eslint-config-standard/blob/63e644c7a2bf464f56afad1324bd88ea9cc6f6a6/package.json#L52-L55